sdk/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+export async function get(url: URL | string) {
+    const response = await fetch(url);
+
+    if (!response.ok) {
+        throw new Error(response.status + " could not get URL " + url);
+    }
+
+    return response;
+}
```

Comment:
Well if we want to use fetch in multiple places in the code, we don't want code duplication, so I create a function in a file, so that way it can just import it and use it. That's normal practice when a function is used multiple times.

By default `fetch` does not throw errors, so every place that uses fetch must implement error handling. So by putting it into a function we don't have to re-implement that behavior multiple times.